plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryFailureRecoveryTest.java

```diff
@@ -73,15 +75,23 @@ protected void testAnalyzeTable()
     @Override
     protected void testDelete()
     {
-        // This connector does not support modifying table rows
-        abort("skipped");
+        // This simple delete on BigQuery ends up as a very simple, single-fragment, coordinator-only plan,
+        // which has no ability to recover from errors. This test simply verifies that's still the case.
+        Optional<String> setupQuery = Optional.of("CREATE TABLE <table> AS SELECT * FROM orders");
+        String testQuery = "DELETE FROM <table> WHERE orderkey = 1";
+        Optional<String> cleanupQuery = Optional.of("DROP TABLE <table>");
+
+        assertThatQuery(testQuery)
+                .withSetupQuery(setupQuery)
+                .withCleanupQuery(cleanupQuery)
+                .isCoordinatorOnly();
     }
 
     @Test
     @Override
     protected void testDeleteWithSubquery()
     {
-        // This connector does not support modifying table rows
+        assertThatThrownBy(super::testDeleteWithSubquery).hasMessageContaining("This connector does not support modifying table rows");
         abort("skipped");
```

Comment:
Great stuff @ebyhr just wondering if this should be removed?

Reply:
Only DELETEs whose predicate can be pushed down are supported.

Reply:
it's expected. the delete needs to operate on physical tables for it to be supported with this code.